app/models/solr_document.rb

```diff
@@ -120,6 +120,20 @@ def file_ids
     self[:img_info] || self[:file_id] || []
   end
 
+  # Checks holdings to see if there are any physical holdings
+  # If there aren't and the library isn't sul, return the library name
+  def library_display
```

Comment:
I think I'm struggling with the method name a little bit; maybe a super explicit name would help?

```suggestion
  def eresources_library_display_name
```